tests/cases/fourslash/unusedParameterInLambda2.ts

```diff
@@ -0,0 +1,14 @@
+/// <reference path='fourslash.ts' />
+
+// @noUnusedLocals: true
+// @noUnusedParameters: true
+//// function f1() {

```

Comment:
Is the outer function needed? Could we reproduce this with just `x => {}`?

Reply:
I just copied the existing test, but I can remove it, if you prefer.

Reply:
Yeah, would be better to simplify the other test too.

Reply:
Done.